cheroot/server.py

```diff
@@ -1805,7 +1793,22 @@ def bind(self, family, type, proto=0):
                 # this machine's TCP stack
                 pass
 
+        # So we can reuse the socket...
+        try:
+            if not IS_WINDOWS and family == socket.AF_UNIX:
```

Comment:
Can we omit the `IS_WINDOWS` check? It feels like a redundant check, especially after having constructed a `socket.socket(AF_UNIX)`. It's also conceivable (though unlikely) that Windows could support this socket type. Better to just let operations fail, I think.

Reply:
As I recall accessing `socket.AF_UNIX` causes `AttributeError` under Windows, that's why this guard exists. Windows doesn't have such a concept at all.